### ISSUE TITLE
Born XHTML & ARIA structural semantics

### ISSUE THREAD

**Issue by LaurenceZaysser** [mattgarrish/epub-revision#2719]
### Section

https://w3c.github.io/epub-specs/epub33/core/#sec-xhtml-req

### Describe the problem

For accessibility sake, it is important to remind that by design, XHTML/HTML5 and ARIA already provide structural semantics used by screen readers for visually impaired. Semantic XHTML tags and ARIA role attribute MUST be used to build born accessible EPUBs. This would help a lot screen readers investing on HTML/ARIA semantic structure. So far (though it might change one day) epub:type is dedicated to display in EPUB reading systems.

XHTML TAGS
```
<section>
<time>
<address>
<q>, <blockquote>
<abbr>
<dfn> 
<dl>, <dt>, <dd>
<code> 
<pre>
<data>
<samp>
<kbd>
<var>

```
(DPUB-)ARIA ATTRIBUTES - ROLES
Report to https://www.w3.org/TR/wai-aria-1.2/#role_definitions
Report to https://www.w3.org/TR/dpub-aria-1.0/
Report to https://www.w3.org/TR/epub-ssv/
Report to https://www.w3.org/TR/epub-aria-authoring-11/

**Comment by mattgarrish** [comment]
We already have an extensive caution note in the epub:type attribute definition to explain this: https://www.w3.org/TR/epub-34/#caution-7

**Comment by LaurenceZaysser** [comment]
Yes, the information is already provided, twice:
- in [C.1 Introduction](https://www.w3.org/TR/epub-34/#sec-structural-semantics-intro)
- in [C2. The type attribute](https://www.w3.org/TR/epub-34/#sec-structural-semantics-intro)

But who will go down there to find it? The spec is hudge. The TOC is much important to help people identify important points, as nobody reads it all in one go...
Maybe the simple renaming of "_C.1 Introduction_" into "_C1. The epub:type and role attributes_" would help?

**Comment by LaurenceZaysser** [comment]
OK for adding a TOC entry named "_Relationship to ARIA-DPUB_" as you suggest. 
Though I am afraid it too restrictive. The relationship to ARIA (roles & states) must be clarified,
as some people still maintain that EPUB accessibility on only defined by EPUB A11Y spec,
and are not concerned by WCAG nor ARIA spec.  Sic ! :-(

**Comment by LaurenceZaysser** [comment]
I am realizing that some EPUB may also contain some interactive content (EPUBs with MCQs typically). This means that ARIA roles are not even sufficent. It is also important to mention ARIA status and properties.
Better having a TOC entry about ARIA to group all this roles + status + properties, no ?!...

**Comment by mattgarrish** [comment]
There's already a scripting accessibility section that refers to WAI-ARIA: https://www.w3.org/TR/epub-34/#sec-scripted-a11y

Sorry, I just don't share your enthusiasm for informative sections in standards. The EPUB specification is not where people go to learn how to make accessible HTML or SVG content. And if someone completely clueless about accessibility lands at the specification, they're going to be looking for these general accessibility sections, not seeking out the nuts and bolts of how things like ARIA get implemented.

**Comment by LaurenceZaysser** [comment]
In private companies, people have less and less time for reading international standards. Even great people with a strong computer science background are misleaded (as they only parse the TOC).
MOST PEOPLE still do not (want to) understand that an EPUB is a puzzle of the best Web standards !
Please help them understanding how the EPUB format is addressing other web standards. Help them making good decisions (technical ones, and human resources ones). So far, in digital publishing, the culture in information science is still a catastropha (as publishers gave the digital power to ignorant multimedia project managers). But the good news is that they still respect the W3C specs. So far... 
HELP! ;-)

**Comment by mattgarrish** [comment]
Closing this issue as changes to the [accessibility section](https://www.w3.org/TR/epub-34/#sec-accessibility) this revision already has identified both [ARIA](https://www.w3.org/TR/wai-aria/) and [ARIA in HTML](https://www.w3.org/TR/html-aria/) as important documents for consideration.

There's also an expanded [additional resources](https://www.w3.org/TR/epub-a11y-12/#sec-a11y-resources) section in the accessibility spec that in addition to those links adds a reference to the [EPUB Type to ARIA Role Authoring Guide](https://www.w3.org/TR/epub-aria-authoring-11/).

There are also external resources for understanding roles and epub:type, such as in the [DAISY knowledge base](https://kb.daisy.org/publishing/docs/html/index.html).

**Comment by iherman** [comment]
This was discussed during the [pmwg meeting on 02 July 2026](https://w3c.github.io/pm-wg/minutes/2026-07-02.html#0896).

<details><summary><i>View the transcript</i></summary>

<h4><a href="https://github.com/w3c/epub-specs/issues/2719" rel="noopener noreferrer">w3c/<wbr>epub-specs#2719</a></h4>
<p><cite>wendyreid:</cite> Great! Next is ARIA structural semantics<br>
<span>… we had some caution in 3.4, is it already addressed?</span></p>
<p><cite>mgarrish:</cite> This is a request to go over html elements with semantics, but this seems outside our scope<br>
<span>… there isn't a clear request here</span><br>
<span>… But how ARIA applies to HTML doesn't make sense for EPUB. There are better resources available</span><br>
<span>… so there isn't much we can do</span></p>
<p><cite>sueneu2:</cite> This is a bigger question. The poster is frustrated with how the spec relates to topology<br>
<span>… This is a bigger question about how we inform the community about what they should do</span></p>
<p><cite>wendyreid:</cite> In this case it is hard, since we don't control the technologies in question</p>
<p><cite>&lt;gpellegrino&gt;</cite> +1 to Matt</p>
<p><cite>mgarrish:</cite> This really belongs to ARIA and HTML, it isn't our domain to get in here. It is up to HTML<br>
<span>… I don't see it fitting in here. Entirely outside our domain.</span><br>
<span>… and ARIA and HTML folks will gety annoyed with us</span></p>
<p><cite>DaleRogers:</cite> It is not part of the spec, but referring them to an explainer? It is like a puzzle and just tell them how it works<br>
<span>… Maybe just point at an explainer or note to help people</span></p>
<p><cite>sueneu2:</cite> Thinking on the kindest and most helpful thing, is it just to close, or is it to add the resources Matt mentioned and leave it open for while?</p>
<p><cite>GeorgeK:</cite> I say we add the DAISY KB links and close it</p>
<p><cite>mgarrish:</cite> Where are we putting them?</p>
<p><cite>GeorgeK:</cite> Just the bug</p>

<hr /></details>

**Comment by LaurenceZaysser** [comment]
If you refuse to add a TOC entry on ARIA in the section [1.3 Relationship to other specifications](https://w3c.github.io/epub-specs/epub33/core/#sec-intro-relations) to help readers and decision-makers, could you at least add a word on ARIA in the section [C. Expressing structural semantics](https://w3c.github.io/epub-specs/epub33/core/#app-structural-semantics)? In my opinion, it would even deserve a C3 section, as ARIA standard is also providing semantics to ATs!

Well structured EPUBs have to address every reading software and every reader. EPUB-HTML authors should know that semantics not only can, but must be indicated in both technical ways (epub:type + ARIA role) in order to produce born accessible EPUBs for visual and not visual readers. You can even add Schema.org microdata if you include SEO in your production goals. 

Double or triple effort then. Good to know if you are considering using an intermediate XML format in your print+EPUB industrial process, with an adequate fine and single semantic structure (specified by publishers) to generate three semantic divergent vocabularies in the final EPUB deliverable. As long as epub:type/ARIA/Schema do not merge...

**Comment by iherman** [comment]
(Admin response only) 

The two links should be set to the EPUB 3.4 version:

- Relationship to other specification is now https://www.w3.org/TR/epub-34/#sec-intro-relations
- Expressing structural semantics is in https://www.w3.org/TR/epub-34/#sec-structural-semantics

This does not affect your argument for the first item, it may for the second: structural semantics is now §7.4.3.1, ie, deeper in the TOC hierarchy

**Comment by mattgarrish** [comment]
As explained on the call yesterday, ARIA is not a unique feature of EPUB and it is not a technology that EPUB adapts in some special way. It is a feature of HTML and its use and processing are defined by that standard (along with the ARIA in HTML document, ARIA AAM, etc.). There is nothing directly that EPUB needs to do to implement either the html role attribute or the ARIA or DPUB-ARIA vocabularies used in it. That is why it is not listed in the relationships section, the same with many other technologies that are similarly inherited from other standards (like microdata, for example). It is also why it doesn't have a section anywhere else, because there is nothing normative that EPUB needs to clarify in order to use the technology. There are plenty of references to ARIA and how it differs from epub:type in the standard already.

In terms of whether ARIA needs to be used to make content accessible, that is also not in scope for the main standard and nothing has specifically been identified for the accessibility one. Requirements for ARIA fall under WCAG success criteria, in particular [SC 4.1.2](https://www.w3.org/TR/WCAG22/#name-role-value) for scripted content.

Whether we should produce a guide to using ARIA/DPUB-ARIA in EPUB would be a more interesting question to look at than what's been proposed in this issue, but that sort of document we'd normally hand off to the CG to work on.

**Comment by sueneu** [comment]
@LaurenceZaysser you speak to a frustration I also feel when accessing the spec. To be honest, I tend to search for my area of interest rather than read the spec sequentially. Secondarily, I check the navigational Table of Contents. Putting the Aria link in the "Relationships to" section at the top would be lost on me. It would be fascinating to know more about how people access the spec, since between the two of us there are already two valid approaches. There is lots more conversation to be had about producing a guide. 

For now, in 3.4, If the information in the [caution note](https://www.w3.org/TR/epub-34/#sec-structural-semantics) had its own entry in the side navigation with the text proposed above:

`7.3.3 The epub:type and role attributes`

Would that be helpful?

**Comment by mattgarrish** [comment]
> 7.3.3 The epub:type and role attributes

The problem with making a caution box into an informative section is that it loses its importance. It becomes just another section that document skimmers will ignore, thereby missing the very important distinction we want to draw between the two attributes.
